### DISCUSSION TITLE
Option: Do not crop media

### DISCUSSION THREAD

**Discussion by Fell** [NickColley/semaphore#31] [New features and ideas]
I follow a lot of people who like to post portrait-mode photos. 

The current available option "Show large inline images and videos" seems to zoom out the image, but still crops it.

Screenshot (Firefox 109)
![grafik](https://user-images.githubusercontent.com/40644/214251379-68e0c89d-2364-4a0b-9c40-d9515527cb05.png)

I propose the option "Do not crop media" (or similar) which will cause the timeline to never crop images, but instead just fit them to timeline width.

This behaviour would also be in line with the default Mastodon UI.

It might be wise to constrain the height to a very high number (like 9999px) to mitigate people trolling with very narrow but tall images. I haven't seen this happen, but it would be possible otherwise.

**Comment by Fell** [comment]
Alternatively, just not cropping images when their aspect ratio is beween 1:2 and 2:1 might be sufficient, like in the upcoming version of Tusky: https://github.com/tuskyapp/Tusky/pull/2832

**Comment by NickColley** [comment]
I'll investigate this more but this feels more like a bug to me.
